### PR TITLE
Dev/responsive pages part1 (PRA-187)

### PR DESCRIPTION
**Pull request by AnnaPestova1**
## One Line Description
responsive saved recipe page and responsive single recipe page
## Requirements

_What this code should do_
added breakpoints for response design
## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Review by ElenaGor8 — APPROVED** [review]
looks good!

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
Great job! Just a couple of comments:

1. Could you please fix page title to align it to the center of the page?
2. Could you please fix the size of alt image text (make it smaller)
3. Could you please try to implement what all text should be inside the box? 
Снимок экрана 2023-08-30 в 20 53 59

**Comment by Ekaterina-Bondareva** [comment]
Single recipe page looks great as it is. Maybe just if you will have a time also align title to the center. 
Снимок экрана 2023-08-30 в 21 10 46

**Comment by AnnaPestova1** [comment]
> Great job! Just a couple of comments:
> 
> 1. Could you please fix page title to align it to the center of the page?

The title is on the center on the Grid with recipes

> 2. Could you please fix the size of alt image text (make it smaller)

fixed the card sizes and text here is Image alt

> 3. Could you please try to implement what all text should be inside the box?

Done
> 
> Снимок экрана 2023-08-30 в 20 53 59

File: src/components/SavedRecipes/SavedRecipesItem.tsx [hunk 1]
```diff
@@ -24,23 +24,32 @@ const SavedRecipesItem = ({ recipe }: SavedRecipeProps) => {
...
-            alt={recipe.recipeName}
-          />
-          <Stack mt="3">
-            <Center>
-              <Text fontSize="xs" h="7">
-                {recipe.recipeName}
-              </Text>
-            </Center>
+    <GridItem
+      // maxW={{ base: "175px", md: "200px" }}
```
Comment (rebekahcallkacz) [inline src/components/SavedRecipes/SavedRecipesItem.tsx:28]: NON-BLOCKING: If you don't need these anymore, I'd just delete them. (see line 35 too)

**Review by rebekahcallkacz — APPROVED** [review]
LGTM

**Review by Ekaterina-Bondareva — APPROVED** [review]

**Pull request merged**